code/modules/tgui/tgui.dm

```diff
@@ -125,7 +125,7 @@
 	// Instruct the client to signal UI when the window is closed.
 	// NOTE: Intentional \ref usage; tgui datums can't/shouldn't
```

Comment:
Should probably change this too since it's not `\ref` anymore.

Reply:
Will do when I can